client-src/elements/chromedash-app.js

```diff
@@ -120,6 +167,16 @@ class ChromedashApp extends LitElement {
     this.contextLink = '/features';
     this.sidebarHidden = true;
     this.selectedGateId = 0;
+    this._clientProvider = new ContextProvider(
+      this,
+      chromestatusOpenApiContext,
+      new Api(
```

Comment:
How would you feel about making this a global variable (like csClient) rather than using lit contexts?

Reply:
I am 50/50 on this. 
I was reading around that this context [pattern](https://lit.dev/docs/data/context/#services) might be the preferred way to pass such a service in lit.

Some things I read:

- [yaqs/6510228263272448](http://yaqs/6510228263272448)
- https://www.patterns.dev/posts/provider-pattern
- [dont-use-window-for-declaring-global-functionsvariables](http://go/js-practices/externs#dont-use-window-for-declaring-global-functionsvariables)
- [discouraged-export-options](https://goto.google.com/typescript-practices/exports#discouraged-export-options)

With all that said, given that it is in lit labs (aka pre stable), I wouldn't be opposed to using the window global now and have an item in the backlog for once it becomes stable.

Let me know your thoughts.

Reply:
I prefer the global variable.  This client is used throughout the app, so it is effectively global.  Removing the context stuff would be one less concept to understand.   If we start using contexts for passing app state around, then we could use it for this too, but I don't see the need for it yet.

Reply:
I have implemented the global variable. Turns out since we are importing it, it cannot follow the same gulp process as the existing cs client. I added notes about it in this commit https://github.com/GoogleChrome/chromium-dashboard/pull/2771/commits/ae39bcd1465ea17165a38210661797fae2ecb50a

Now there's a new gulp task `rollup-cjs`  to convert it from the ES module to CJS. Then it can be used like the existing cs client.